### PR TITLE
video2midi: init at 0.3.7

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change

Youtube synthesia video to midi conversion tool

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by gnidorah** [comment]
Why grahamcofborg-eval fails? @grahamc

File: pkgs/tools/audio/video2midi/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  pythonPath = with python2Packages; [ opencv3_ midiutil pygame pyopengl ];
+  nativeBuildInputs = with python2Packages; [ python wrapPython ];
+
+  installPhase = ''
+    install -Dm755 v2m.py $out/bin/v2m.py
+    wrapPythonPrograms
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Youtube synhesia video to midi convesion tool";
```
Comment (alexarice) [inline pkgs/tools/audio/video2midi/default.nix]: synthesia, conversion

**Review by alexarice — APPROVED** [review]
Looks good. I have run nix-review and tested the resulting program and it seems to work well.

**Comment by gnidorah** [comment]
@alexarice Corrected, thanks!

**Comment by lheckemann** [comment]
@grahamcofborg build video2midi

**Comment by lheckemann** [comment]
What will they come up with next?

**Comment by gnidorah** [comment]
@lheckemann :smile: Thank you!

**Pull request merged**